CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Updates should follow the [Keep a CHANGELOG](http://keepachangelog.com/) princip
 
 * Update nette libs to version 3.0.0 (BC break)
 * Pretty JSON output in API console - without escaping unicode and slashes
+* Replaced handler information array triplet (endpoint, handler, authorization) with HandlerSettings
```

Comment:
iba uvazujem - ten nazov HandlerSettings - keby sme to volali len ze "Api" ? :-) alebo nejaka ina idea ako HandlerSettings?

Reply:
:) sak aj ja som napisal ze s tym nazvom nie som uplne stotozneny